test/neon_test_resizable_pages.dart

```diff
@@ -0,0 +1,35 @@
+@HtmlImport('neon_test_resizable_pages.html')
```

Comment:
Can you add a `TODO(jakemac): Add NeonSharedElementAnimatableBehavior and IronResizableBehavior to all of these once https://github.com/dart-lang/polymer-dart/issues/551 is resolved.`